index.js

```diff
@@ -267,45 +300,53 @@ module.exports = {
     // defined using variables (e.g. when created using a builder).
     "no-use-before-define": "off",
 
-    // ----- Node.js and CommonJS -----
-
-    // Disabled because this is just guessing.
-    "callback-return": "off",
-
-    // Disabled because one occasionally needs a non-global `require` call, e.g.
-    // when using an optional dependency, choosing from multiple alternative
-    // dependencies, or implementing a plugin system.
-    "global-require": "off",
-
-    // Disabled because this is just guessing.
-    "handle-callback-err": "off",
-
-    // Enabled, however this isn't strictly necessary because `one-var` doesn't
-    // allow multiple `require` calls in one `let`/`const`/`var` statement. But
-    // it can't hurt.
-    "no-mixed-requires": "off",
-    "no-new-require": "error",
-
-    // Disabled because code using `path.join` is usually more verbose than
-    // string concatenation or template literals. In theory, `path.join` is more
-    // portable because it always uses correct path separator, but since Node.js
-    // on all platforms can deal with paths that use `/`, this is not really an
-    // advantage.
-    "no-path-concat": "off",
-
-    // Disabled because `process.env` has legitimate uses, mainly in binaries.
-    "no-process-env": "off",
-
-    // Disabled because `process.exit` has legitimate uses, mainly in binaries.
-    "no-process-exit": "off",
-    "no-restriced-modules": "off",
-    "no-sync": "off",
+    // ----- Stylistic Issues -----
+
+    "array-bracket-newline": [ "error", "consistent" ],
+    "array-bracket-spacing": [ "error", "always", { objectsInArrays: false }],
```

Comment:
I'm ok with that, but I want `object-curly-spacing` to match.

Reply:
I'm sure they don't have to match. Object literals often a quite long if written in the one string because of property name and property value, and they looks quite packed without additional spaces; the arrays, in the contrary, usually a small and additional spaces makes them too spread (well, at least in Peggy it seems be true). So I think that `"object-curly-spacing": ["always"]`, but the `"array-bracket-spacing": ["never"]` is a reasonable choice.

If you disagree and think that they should match, then I prefer to keep `"always"` in both.

Reply:
After looking at what changed in the patch for a while, I'm ok with them being different.  I'll make the change, and ensure that the .ts rules match their .js counterparts; `object-curly-spacing` is currently different in each for example.